src/github/pullRequestManager.ts

```diff
@@ -939,21 +947,21 @@ export class PullRequestManager implements vscode.Disposable {
 	}
 
 	async getIssueComments(pullRequest: PullRequestModel): Promise<Octokit.IssuesListCommentsResponseItem[]> {
-		Logger.debug(`Fetch issue comments of PR #${pullRequest.prNumber} - enter`, PullRequestManager.ID);
+		Logger.debug(`Fetch issue comments of PR #${pullRequest.githubNumber} - enter`, PullRequestManager.ID);
 		const { octokit, remote } = await pullRequest.githubRepository.ensure();
 
 		const promise = await octokit.issues.listComments({
 			owner: remote.owner,
 			repo: remote.repositoryName,
-			issue_number: pullRequest.prNumber,
+			issue_number: pullRequest.githubNumber,
 			per_page: 100
 		});
-		Logger.debug(`Fetch issue comments of PR #${pullRequest.prNumber} - done`, PullRequestManager.ID);
+		Logger.debug(`Fetch issue comments of PR #${pullRequest.githubNumber} - done`, PullRequestManager.ID);
 
 		return promise.data;
 	}
 
-	async createIssueComment(pullRequest: PullRequestModel, text: string): Promise<IComment> {
+	async createIssueComment(pullRequest: IssueModel, text: string): Promise<IComment> {

```

Comment:
nit: `issue: IssueModel`

Reply:
or perhaps `item: IssueModel | PullRequestModel`? might be clearer

Reply:
Changed to issue